docs/Consensus-Protocols/IBFT.md

```diff
@@ -77,6 +53,48 @@ Properties that have specific values in IBFT 2.0 genesis files are:
 
 To start a node on an IBFT 2.0 private network, use the [`--genesis-file`](../Reference/Pantheon-CLI-Syntax.md#genesis-file`) option to specify the custom genesis file. 
 
+### Extra Data 
+
+The `extraData` property is RLP encoded. RLP encoding is a space efficient object serialization scheme 
+used in Ethereum. Use the Pantheon subcommand [`rlp encode`](../Reference/Pantheon-CLI-Syntax.md#rlp) 
+to generate the `extraData` RLP string to include in the genesis file. 
+
+!!! example                                        
+    ```bash
+    pantheon rlp encode --from=toEncode.json --to=rlpEncodedExtraData
+    ```    
+
+Where the `toEncode.json` file contains a list of the initial validators in ascending order. 
+
+!!! example "One Initial Validator"
+    ```json
+    [
+     "9811ebc35d7b06b3fa8dc5809a1f9c52751e1deb"
+    ]
+    ``` 
+
+Copy the RLP encoded data from the `rlpEncodedExtraData` file to the `extraData` in the genesis file. 
+
+### Block Time 
+
+When a block is received, the block time and timeout timers are started. When `blockperiodseconds` is reached, 
+a new block is proposed. Generally, the proposed block is added before reaching `requesttimeoutseconds`. A new 
+round is then started, and the block time and timeout timers are reset. When `blockperiodseconds` is reached, the next 
+new block is proposed. 
+
+If `requesttimeoutseconds` is reached before the proposed block is added, a round change occurs, and the block time and 
```

Comment:
This sentence need to appear before the above "..Generally, the proposed block ...."